resources/assets/templates/vote/voteHeader.hbs

```diff
@@ -34,6 +34,27 @@
                 </ul>
             </div>
         {{/if}}
+
+        {{#if meta.isFinished}}
+        {{else}}
+        <div>
+            {{#if model.summary}}
+            <div><hr><h3>{{model.summary}}</h3><hr></div>
+            {{else}}
+            <div><hr><h3>No summary yet</h3><hr></div>
```

Comment:
No necessary to show 'No summary yet'. Just show nothing if no summary presented.